sensor/tests/resource/service/yaml/nginx-service-node-port-30007.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: nginx-svc-node-port
+  name: nginx-svc-node-port-30007
```

Comment:
This is very annoying. Maybe not for this PR if this generates some CI failures, but it would be great to have a template for this other than having to copy the file content.

Or, we could also offer the possibility to create resources using Go structs as opposed to yaml files.

Reply:
My thoughts exactly, but I think we should do that in a different PR. I wanted to time box this fix to focus in other tasks.